kyuubi-server/src/main/scala/org/apache/kyuubi/server/mysql/MySQLCommandHandler.scala

```diff
@@ -53,6 +55,8 @@ class MySQLCommandHandler(be: BackendService, execPool: ThreadPoolExecutor)
 
   // handle process exception, generally should send error packet
   override def exceptionCaught(ctx: ChannelHandlerContext, cause: Throwable): Unit = {
+    MetricsSystem.tracing(_.incCount(MYSQL_CONN_FAIL))
```

Comment:
move to L64, only `MetricsSystem.tracing(_.incCount(MYSQL_CONN_FAIL))`  if `ctx.channel.isActive`?

cc @pan3793